<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VirshGui</class>
 <widget class="QMainWindow" name="VirshGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VirshGui</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="serverAddrLabel">
        <property name="text">
         <string>Server Adresse</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="hostEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="portLabel">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="portEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="userLabel">
        <property name="text">
         <string>Nutzer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="userEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="passwordLabel">
        <property name="text">
         <string>Passwort</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="passwordEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="connectButton">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTableView" name="vmListTable">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="QTextBrowser" name="vmInfoText">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>937</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
